src/main/java/org/apache/accumulo/access/AccessEvaluatorImpl.java

```diff
@@ -30,7 +30,7 @@
 import java.util.stream.Stream;
 
 //this class is intentionally package private and should never be made public
-class AccessEvaluatorImpl implements AccessEvaluator {
+public class AccessEvaluatorImpl implements AccessEvaluator {
```

Comment:
This change conflicts with the comment right above it.

Reply:
Yes, I can fix by making a method on the interface that calls this method as is done with some of the other methods. I wanted @keith-turner to look at this first though to be sure that I got the escaping/quoting correct in AccessExpressionEvaluator.